app/backend/src/clj/teet/migration/activity_procurement_info.clj

```diff
@@ -21,3 +21,16 @@
                           {:activity/procurement-id procurement-id})
                         (when procurement-nr
                           {:activity/procurement-nr procurement-nr})))})))
+
+(defn retract-procurement-info-from-activity [conn]
+  (let [activities (d/q '[:find ?a
+                          :where [?a :activity/procurement-id _]
+                          [?a :activity/procurement-nr _]]
+                     (d/db conn))]
+    (d/transact conn
+      {:tx-data (for [[activity-id] activities]
+                  [:db/retract activity-id :activity/procurement-id])})
+
+    (d/transact conn
```

Comment:
why is this two separate transactions instead of just one? And shouldn't the retract be given the attribute value that is being retracted as well https://docs.datomic.com/cloud/tutorial/retract.html ?

Reply:
Two transactions implemented to avoid mergeing two lists for one transaction. Could probably be re-factored into one.
My understanding of this tutorial that we can add attribute value to be retracted, only if we care about other attributes values, which we want to keep. In this case we want to retract all values of procurement-id, and procurement-nr, don't we?